x/gamm/keeper/pool_service_test.go

```diff
@@ -751,6 +751,45 @@ func (suite *KeeperTestSuite) TestJoinSwapExactAmountInConsistency() {
 	}
 }
 
+func (suite *KeeperTestSuite) TestGetPoolDenom() {
+	// setup pool with denoms
+	suite.FundAcc(suite.TestAccs[0], defaultAcctFunds)
+	poolCreateMsg := balancer.NewMsgCreateBalancerPool(suite.TestAccs[0], defaultPoolParams, defaultPoolAssets, defaultFutureGovernor)
+	_, err := suite.App.GAMMKeeper.CreatePool(suite.Ctx, poolCreateMsg)
+	suite.Require().NoError(err)
+
+	for _, tc := range []struct {
+		desc         string
+		poolId       uint64
+		expectDenoms []string
+		expectErr    bool
+	}{
+		{
+			desc:         "Valid PoolId",
+			poolId:       1,
+			expectDenoms: []string{"bar", "foo"},
+			expectErr:    false,
+		},
+		{
+			desc:         "Invalid PoolId",
+			poolId:       2,
+			expectDenoms: []string{"bar", "foo"},
+			expectErr:    true,
+		},
+	} {
+		suite.Run(fmt.Sprintf("Case %s", tc.desc), func() {
```

Comment:
Might be a personal preference here so feel free to ignore if you disagree - I see little value in the Case prefixes and I feel like they are only making it harder to identify distinct cases printed in the command line, especially when there are many cases
```suggestion
		suite.Run(tc.desc, func() {
```

Reply:
Please ping me once you decide whether to address it, and I will merge this PR :+1:

Reply:
oh yea i can add that. I dont have a strong opinion on it